File: Factorial.cpp
//write a program to find factorial 

#include<iostream>
using namespace std;

int findFactorial(int iNo)
{
   int iFact=1;
   for(int i=1;i<=iNo;i++)
   {
    iFact=iFact*i;
   }
   return iFact;
}
int main()
{
    int iNo=0;

    cout<<"Enter the number"<<endl;
    cin>>iNo;

   int iRet=0;

    iRet=findFactorial(iNo);

    cout<<"Factorial of"<<iNo << "is" << iRet <<endl;

    return 0;
}

File: Factorial.java
import java.util.Scanner;

class Demo
{
    int findFactorial(int iNo)
    {
        int iFact=1;

        for(int i=1;i<=iNo;i++)
        {
            iFact=iFact*i;
        }
       return iFact;
    }
}
class Factorial
{

    public static void main(String args[])
    {
        Scanner sobj=new Scanner(System.in);
        int iNo=0;

        System.out.println("Enter the number");

        iNo=sobj.nextInt();

        Demo dobj=new Demo();
        int iRet=dobj.findFactorial(iNo);

        System.out.println("Factorial of" + iNo+ "is" +iRet);
       
       sobj.close();
    }
}